Makefile

```diff
@@ -0,0 +1,16 @@
+init:
```

Comment:
is there a reason not to use the same makefile as in other node implementations, like e.g.: https://github.com/ProxeusApp/node-mail-sender/blob/master/Makefile

Reply:
changed